common/lostfiles.in

```diff
@@ -39,127 +35,197 @@ while getopts 'sz' OPTION; do
+    "/etc/resolv.conf.bak" # networkmanager - automatically created DNS server list backup
+    "/etc/shadow"
+    "/etc/shadow-"
+    "/etc/shadow.OLD"
+    "/etc/texmf/ls-R" # somewhere inside texlive-most - directory contents cache
+    "/etc/texmf/web2c/updmap.cfg"  # somewhere inside texlive-most - autogenerated font maps file
+    "/etc/udev/hwdb.bin"
+    "/etc/ssh/ssh_host*"
+    "/etc/NetworkManager/system-connections"
+    "/etc/rc.digitalocean"
+    "/etc/samba/private/passdb.tdb"
+    "/etc/samba/private/secrets.tdb"
+    "/etc/samba/private/smbpasswd"
+    "/etc/samba/smb.conf"
+    "/etc/ssl"
+    "/etc/xml"
+    "/etc/systemd/system"
+    "/etc/systemd/network"
+    "/etc/systemd/user" # systemd - User defined services
+    "/etc/udev/rules.d/70-digitalocean-net.rules"
+    "/etc/vconsole.conf"
+    "/etc/zfs/zpool.cache"
+    "/home"
+    "/incoming"
+    "/lost+found"
+    "/media"
+    "/mnt"
+    "/opt/google-appengine-go/appengine/api/*.pyc"
+    "/proc"
+    "/root"
+    "/run"
+    "/scratch"
+    "/srv"
+    "/sys"
+    "/tmp"
+    "/usr/bin/__pycache__"
+    "/usr/lib/gdk-pixbuf-2.0/*/loaders.cache" # gdk-pixbuf2 - Cache files
+    "/usr/lib/gio/modules/giomodule.cache" # gvfs? - Cache file
+    "/usr/lib/gtk-2.0/*/immodules.cache" # gtk2? - Cache files
+    "/usr/lib/gtk-3.0/*/immodules.cache" # gtk3? - Cache files
+    "/usr/lib/libnetbrake.so.0" # netbrake - Symlink to DLL
+    "/usr/lib/locale/locale-archive"
+    "/usr/lib/modules/$CURRENTKERNEL"
+    "/usr/lib/udev/hwdb.bin" # systemd - Hardware database generated by systemd-hwdb
+    "/usr/lib/virtualbox/ExtensionPacks/Oracle_VM_VirtualBox_Extension_Pack"
+    "/usr/local"
+    "/usr/share/dict/words"
+    "/usr/share/glib-2.0/schemas/gschemas.compiled" # glib2 - Cache files
+    "/usr/share/hplip/**/__pycache__" # hplip - Python cache files
+    "/usr/share/icons/*/icon-theme.cache" # ??? - Cache file
+    "/usr/share/info/dir"
+    "/usr/share/.mono" # mono - Intermediate dir
+    "/usr/share/.mono/certs" # mono - Intermediate dir
+    "/usr/share/.mono/new-certs" # mono - Intermediate dir
+    "/usr/share/.mono/certs/Trust" # mono - ???
+    "/usr/share/.mono/new-certs/Trust" # mono - ???
+    "/usr/share/applications/mimeinfo.cache"
+    "/usr/share/backintime"
+    "/usr/share/fonts/.uuid" #  ??? - ???
+    "/usr/share/fonts/*/.uuid" # ??? - ???
+    "/usr/share/fonts/*/fonts.dir" # ??? - ???
+    "/usr/share/fonts/*/fonts.scale" # ??? - ???
+    "/usr/share/mime"
+    "/usr/share/texmf/ls-R"  # somewhere inside texlive-most - directory contents cache
+    "/usr/share/texmf-dist/ls-R"  # somewhere inside texlive-most - directory contents cache
+    "/usr/share/webapps/owncloud/data"
+    "/usr/var/cache"
+    "/usr/var/run"
+    "/var/.updated"
+    "/var/lost+found"
+    "/var/abs"
+    "/var/cache"
+    "/var/lock"
+    "/var/lib"
+    "/var/log"
+    "/var/run"
+    "/var/spool"
+    "/var/db/sudo"
+    "/var/tmp"
+  )
```

Comment:
Same here, a lot of those files do not have to be excluded in the first place if you do not search for paths outside the ones I’ve listed before.